src/System.ComponentModel.TypeConverter/tests/BindingListTests.cs

```diff
@@ -0,0 +1,699 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// See the LICENSE file in the project root for more information. 
+
+using System;
+using System.ComponentModel;
+using System.ComponentModel.Design.Serialization;
+using System.Collections.Generic;
+using Xunit;
+using System.Reflection;
+
+namespace System.ComponentModel.Tests
+{
+	public class BindingListTest
+	{
+		[Fact]
+		public void BindingListDefaults ()
```

Comment:
Can we cleanup the formatting to match the CoreFX coding style? Mono code style places a space before parens, which would ideally be removed here. Applies throughout.